src/App.tsx

```diff
@@ -34,10 +35,24 @@ export function getReorderedGoods(
   // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
+  visibleGoods.sort((a, b) => {
+    switch (sortType) {
+      case SortType.ALPHABET:
+        return a < b ? -1 : 1;
```

Comment:
I think it's better to use `localeCompare` for strings comparing.

Reply:
agree, fix @Anastasiia-Khudych comment